env.example

```diff
@@ -0,0 +1 @@
+VITE_API_BASE_URL=
```

Comment:
- [ ] Why is this variable equal to nothing?

Reply:
@AndriiKasian The env.example file shows the environment variables we need, but it doesn’t include values. The empty VITE_API_BASE_URL= line helps developers know which variable they should add to their own .env file.